paparazzi/src/main/java/app/cash/paparazzi/internal/resources/ModuleResourceRepository.kt

```diff
@@ -43,7 +43,8 @@ internal class ModuleResourceRepository private constructor(
       resourceDirectories: List<File>
     ): LocalResourceRepository {
       return ModuleResourceRepository(
-        displayName = "", // TODO
```

Comment:
`An anti-bikeshedding Kotlin linter with built-in formatter`

yea....

let's disable this rule in .editorconfig

```
ktlint_standard_discouraged-comment-location = disabled
```

Reply:
btw, I tried doing this locally and came across the error mentioned here:
https://github.com/pinterest/ktlint/issues/2338#issuecomment-1788731817

That looks to be fixed here: https://github.com/pinterest/ktlint/pull/2371
but set to be released in 1.1, so I think this PR will likely hang out (or maybe we can merge some of the changes without bumping to 1.0?) since the "discouraged comment location" is more opinionated than I like.

Reply:
This PR does NOT bump the ktlint version, that's the point. But I'll have to retry some of this it seems based on your feedback with the intellij style re-applied.

Reply:
ok, sounds good.  i can also continue work on this effort, if that's easier for you.